pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -247,3 +274,75 @@ func (db *DB) DeleteTTLByID(id string) error {
 	}
 	return err
 }
+
+// AddMIB adds MIB into database by date
+func (db *DB) AddBwUsageTbl(size, unixtimenow int64) (err error) {
+	defer db.locked()()
+	t := time.Now()
+	fmt.Println("time now =", t.Unix())
```

Comment:
Don't use fmt.Println in code.

Reply:
It was a dev debug info. used log.printf(ln). All dev debug info print outs are cleaned up